app/models/pame_evaluation.rb

```diff
@@ -2,21 +2,12 @@
 require 'wcmc_components'
```

Comment:
Getting issues when running `Wdpa::PameImporter.import`, with or without a file passed, looks like the removal of the wcmc_components gem caused it. Managed to run it successfully with the line removed.

Reply:
Ah well spotted! Forgot to remove this line; will do now. Thanks!